### ISSUE TITLE
[UI] Importing Images as a Sequence Issue

### ISSUE THREAD

**Issue by dloiola** [olive-editor/olive#1193]
**Commit Hash**
007b9f2

**Platform**
Arch Linux - 5.8.7-zen1-1-zen

**Summary**
When importing images that seems to be a sequence, Olive is asking once per image if you want to import as a Sequence.  Clicking either "yes" or "no" it behaves the same way, and clicking "yes" it is importing each image as a separate sequence rather than only one. 

**Additional Information / Output**
[DEBUG] Extracting default OCIO config to "/tmp/olivevideoeditor.org/Olive/ocioconf" ((null):0)
[INFO] Using Qt version: 5.11.0 ((null):0)
[DEBUG] Using graphics driver: Intel Open Source Technology Center ((null):0)
[INFO] Setting output audio device to "alsa_output.pci-0000_00_1b.0.analog-stereo" ((null):0)
[WARNING] QXcbConnection: XCB error: 3 (BadWindow), sequence: 2591, resource id: 16548993, major code: 40 (TranslateCoords), minor code: 0 ((null):0)

**Comment by ThomasWilshaw** [comment]
Hi, please submit a commit hash for this issue. Thanks

**Comment by dloiola** [comment]
Hi, thomas! Sorry for my newbieness but how do a get this hash? Reading the "Read Before..." found no topic about hash.

**Comment by ThomasWilshaw** [comment]
The hash is a commit hash of the version you are using. If you build it locally, it will be the hash of the commit you built. If you downloaded it pre-built, it will probably appear in the filename or the About box. Try to answer the other lines too.

Sorry to be strict, but the last time the issue tracker was opened, there were hundreds of reports that were creating work. The trick to a good report is to do all of the work in advance so a volunteer can spend a few minutes to make the issue easier to track and ultimately solve.

Thanks

**Comment by dloiola** [comment]
Thomas, you don't need to apologise, that's my fault. Updated the Issue text, can you tell if now that's right? I open the Issue #1194 too and would like to know if it ok.

**Comment by ThomasWilshaw** [comment]
Yes that's correct, thanks very much. Looks like the issue has been confirmed too.

**Comment by ThomasWilshaw** [comment]
@dloiola can you confirm if the latest build fixes your issue so we can close this.

**Comment by dloiola** [comment]
With the commit 871bc08c the issue still occurs. I did a `git pull` to the fec97749 commit and will build again to check it out.

**Comment by ThomasWilshaw** [comment]
Thanks, either that or try one of the prebuilt ones.

**Comment by dloiola** [comment]
The build process is almost done

**Comment by dloiola** [comment]
Tested with a sequence of 10 images: 

1. **If I import a single image between 1st and 9th**
     1. _Olive asks for Importing as Sequence only once and proceeds, as expected, generating only one sequence._
 
2. **If I select all the 10 images**
    1. _Olive asks 9 times for Importing as Sequence, no matter you click "yes" or "no"._
    2. _It generates 9 Sequences using all the 10 images._
    3. _It also imports the 10th image as a single image_

I think it should ask only once, clicking "yes" generate a single Sequence, clicking "no" import all as simple images.

![image](https://user-images.githubusercontent.com/20931515/94622621-e7a39f80-0288-11eb-9201-e43ddd278020.png)

**Comment by itsmattkc** [comment]
@dloiola The fix is in the `otio` branch, it hasn't been merged into `master` yet (hence why the "fixes" keyword didn't close this issue automatically).

**Comment by ThomasWilshaw** [comment]
Sorry I missed that was in the other branch. It does seem like some of the changes you've made in `master` have improved importing image sequences though.

**Comment by dloiola** [comment]
I have to just wait for news? Can I help with something for awhile?

**Comment by itsmattkc** [comment]
You can try the branch yourself with `git checkout otio` of you'd like
